### PR TITLE
items created

### PR DESCRIPTION
**Pull request by asim516**

### CODE REVIEW

File: src/main/java/training/TableCreator.java [hunk 1]
```diff
@@ -1,49 +1,69 @@
...
-import com.amazonaws.regions.Regions;
-import com.amazonaws.services.dynamodbv2.AmazonDynamoDB;
-import com.amazonaws.services.dynamodbv2.AmazonDynamoDBClient;
-import com.amazonaws.services.dynamodbv2.document.DynamoDB;
-import com.amazonaws.services.dynamodbv2.document.Table;
-import com.amazonaws.services.dynamodbv2.model.*;
-
-import java.util.Arrays;
-
-public class TableCreator {
```
Comment (CliqueAdmin) [inline src/main/java/training/TableCreator.java:14]: Sorry I did not ask you to create table. Manualy creat table and just write the code to insert items

File: src/main/java/training/CreateItems.java [hunk 2]
```diff
@@ -0,0 +1,73 @@
+package training;
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBAttribute;
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBHashKey;
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBTable;
+
+import java.util.Objects;
+import java.util.Set;
+
+@DynamoDBTable(tableName = "igh")
```
Comment (CliqueAdmin) [inline src/main/java/training/CreateItems.java:9]: Table name is IGHItems sorry I was not clear in my comment.

File: src/main/java/training/TableCreator.java [hunk 3]
```diff
@@ -1,49 +1,69 @@
...
-
-
-    public void createTable(String accessId,String accessSecret){
```
Comment (CliqueAdmin) [inline src/main/java/training/TableCreator.java:17]: Do not delete this tablecreator let it be there this was your old assignment. As I said do not assume things that I asked you to delete this code.

**Comment by CliqueAdmin** [comment]
I will say create a new pull request delete this one.

**Review by CliqueAdmin — CHANGES_REQUESTED** [review]
I would say start with another pull request.

Assume you have IGHItems table created , your code should only insert items in that. 

Do not modify any existing code rather add new code.

Also this time you need to write unit tests for this code.

File: src/main/java/training/TableCreator.java (hunk 1 above)
Comment (asim516) [reply src/main/java/training/TableCreator.java:14]: but the method create() is intserting items

Comment (CliqueAdmin) [reply src/main/java/training/TableCreator.java:14]: First of all why it is in tablecreator class , you need to create separate classes

Comment (CliqueAdmin) [reply src/main/java/training/TableCreator.java:14]: First of all why it is in tablecreator class , you need to create separate classes

Comment (asim516) [reply src/main/java/training/TableCreator.java:14]: by manually create table u mean to say directly from aws website?

**Pull request closed without merging**